src/ontology/uberon-edit.obo

```diff
@@ -7238,27 +7238,15 @@ relationship: present_in_taxon NCBITaxon:4751 ! Fungi
 [Term]
 id: UBERON:0000477
 name: anatomical cluster
-def: "Anatomical group that has its parts adjacent to one another." [http://orcid.org/0000-0001-9114-8737]
+def: "Anatomical group whose component anatomical structures lie in close proximity to each other." [FBbt:00007277]
 subset: common_anatomy
 subset: upper_level
-xref: AAO:0010009
-xref: AEO:0000041
-xref: BILA:0000041
-xref: CARO:0000041
 xref: EHDAA2:0003041
```

Comment:
Could not find this in EHDAA2 using OLS. Should it be removed as well?

Reply:
I can’t find it through Ontobee or BioPortal as well, so I agree, I’ll remove it.